### ISSUE TITLE
Option to change Margin between pages when importing multi-page pdf 

### ISSUE THREAD

**Issue by LeSnake04** [flxzt/rnote#724]
**Is your feature request related to a problem? Please describe.**  
There Is a fixed size margin between pages when importing multi-page PDFs, that can be really annoying when lining them up with pages.

**Describe the solution you'd like**  
Allow users to set the margin between pages.

**Comment by LeSnake04** [comment]
Yes. but sometimes you want 2 pages per Rnote page or need to line it up with other lines.
While I cant remember a specific example right now, I am sure there are PDFs that have images split between pages that need no spacing (especially when looking at flyers). 

Some users may also just don't like the spacing or want a bigger/smaller one, so just  having option to set the value would be a nice to have for them.

**Comment by Kneemund** [comment]
What unit would that margin be in? If it's an absolute value, this might become quite unintuitive to get exactly what you want, especially "lining imports up with pages" sounds like a lot of trial and error is necessary.

**Comment by LeSnake04** [comment]
I think it should use the selector used here. It's the most intuitive IMO. 
![image](https://github.com/flxzt/rnote/assets/47243761/7fc1e627-8c6e-4857-91b9-692ca7c649be)

**Comment by LeSnake04** [comment]
Also, we should consider making a margin switch and a separate margin size field, since I assume some user might not realize that they can turn it off by setting it to 0
